test/unit/SwapOperator/closeOrder.js

```diff
@@ -127,36 +94,34 @@ describe('closeOrder', function () {
     await swapOperator.placeOrder(contractOrder, orderUID);
   });
 
-  describe('access control', function () {
-    it('before deadline, its callable only by controller', async function () {
-      const deadline = order.validTo;
-      const snapshot = await takeSnapshot();
+  it('before deadline, its callable only by controller', async function () {
+    const deadline = order.validTo;
+    const snapshot = await takeSnapshot();
 
-      // Executing as non-controller should fail
-      await setNextBlockTime(deadline);
-      await expect(swapOperator.connect(governance).closeOrder(contractOrder)).to.be.revertedWith(
-        'SwapOp: only controller can execute',
-      );
+    // Executing as non-controller should fail
+    await setNextBlockTime(deadline);
+    await expect(swapOperator.connect(governance).closeOrder(contractOrder)).to.be.revertedWith(
+      'SwapOp: only controller can execute',
+    );
 
-      // Executing as controller should succeed
-      await revertToSnapshot(snapshot);
-      await setNextBlockTime(deadline);
-      await swapOperator.connect(controller).closeOrder(contractOrder);
-    });
+    // Executing as controller should succeed
+    await revertToSnapshot(snapshot);
+    await setNextBlockTime(deadline);
+    await swapOperator.connect(controller).closeOrder(contractOrder);
+  });
 
-    it('after deadline, its callable by anyone', async function () {
-      const deadline = order.validTo;
-      const snapshot = await takeSnapshot();
+  it('after deadline, its callable by anyone', async function () {
+    const deadline = order.validTo;
+    const snapshot = await takeSnapshot();
 
-      // Executing as non-controller should succeed
-      await setNextBlockTime(deadline + 1);
-      await swapOperator.connect(governance).closeOrder(contractOrder);
+    // Executing as non-controller should succeed
```

Comment:
let's make this address some random address not governance so the test is accurate